### PR TITLE
[FIXED JENKINS-25666] Adopt for new jenkins core UI

### PR DESCRIPTION
**Pull request by KostyaSha**
Before change 
screenshot 2015-03-14 18 05 35

After change
screenshot 2015-03-14 18 03 49

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -351,6 +351,14 @@ tr.spacerRow td {
     display: none;
 }
 
+#main-panel {
+    margin-left: 15px;
+}
+
+.fixedGridLayout #main-panel {
+    padding-left: 0;
```
Comment (KostyaSha) [inline src/main/webapp/css/main.css]: removes enforced from new core indent in 370px;

File: src/main/webapp/css/main.css [hunk 2]
```diff
@@ -351,6 +351,14 @@ tr.spacerRow td {
     display: none;
 }
 
+#main-panel {
+    margin-left: 15px;
```
Comment (KostyaSha) [inline src/main/webapp/css/main.css]: set left margin the same as right

**Comment by KostyaSha** [comment]
@patbos @tfennelly @daniel-beck review

**Comment by KostyaSha** [comment]
screenshot 2015-03-14 18 03 49

**Comment by daniel-beck** [comment]
@KostyaSha Please stop pinging me about things I'm not involved in and have no interest in. General issue cleanup does not indicate interest. I don't even use the plugin.

**Comment by KostyaSha** [comment]
@daniel-beck ok, but i definitely remember that you took part in changing jenkins UI that now ends with regressions between minor commits.

**Comment by daniel-beck** [comment]
@KostyaSha I doubt that I'm the author of even a single commit related to that. I tested the PR builds and provided feedback on those. And that was half a year ago.

**Comment by patbos** [comment]
:+1:

**Comment by dvankampen** [comment]
I am seeing this change cause the pipeline dashboard view the lay behind the sidebar.  The side-panel was fixed at 370px, and with this setting overriding the main-panel offset, the main-panel is now behind the side panel, rendering it unreadable.

**Comment by KostyaSha** [comment]
@dvankampen We tested this change on new and old jenkins.  Report bug, provide logs, screenshots etc.

**Pull request merged**